### PR TITLE
Add manpage for ansible-console. (Closes: #16244)

### PR DESCRIPTION
**Pull request by hlieberman**
##### ISSUE TYPE
- Bugfix Pull Request
##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```
##### SUMMARY

Add a basic manpage for ansible-console, based heavily off of the ansible manpage.

### CODE REVIEW

File: docs/man/man1/ansible-console.1 [hunk 1]
```diff
@@ -0,0 +1,304 @@
+'\" t
```
Comment (bcoca) [inline docs/man/man1/ansible-console.1]: this file is not needed, just the .in file

**Comment by hlieberman** [comment]
@bcoca, compiled manpage removed and repushed.

File: docs/man/man1/ansible-console.1.asciidoc.in [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+:man manual:   System administration commands
+
+NAME
+----
+ansible-console - a REPL for ad-hoc ansible tasks
+
+
+SYNOPSIS
+--------
+ansible-console <host-pattern> [-m module_name] [-a args] [options]
```
Comment (bcoca) [inline docs/man/man1/ansible-console.1.asciidoc.in]: host-pattern is optional

**Comment by dharmabumstead** [comment]
Looks good to me, but needs a tech review sign off.

File: docs/man/man1/ansible-console.1.asciidoc.in (hunk 2 above)
Comment (hlieberman) [reply docs/man/man1/ansible-console.1.asciidoc.in]: Fixed in 1e158c6.

**Pull request merged**